prime-router/Dockerfile

```diff
@@ -9,7 +9,6 @@ FROM mcr.microsoft.com/azure-functions/java:3.0-java$JAVA_VERSION-build AS insta
 COPY . /src/java-function-app
 RUN cd /src/java-function-app && \
     mkdir -p /home/site/wwwroot && \
-    mvn clean package && \
     cd ./target/azure-functions/ && \
```

Comment:
I'm removing the Java build step for two reasons: 1) much faster to build the image 2) difficulty connecting to local database.

Reply:
Cool - is much faster